src/vinyldns/serdes.py

```diff
@@ -88,9 +88,20 @@ def to_json_string(o):
     :param o: An object that can be serialized to json
     :return: A json formatted string representation of the object
     """
-    import json
     return json.dumps(to_dict(o))
 
+def to_review_json(s):
```

Comment:
put this in the `batch_change.py` ... the `serdes.py` file is only concerned with general serialization and deserialization, this is batch change specific